### ISSUE TITLE
Restoring configuration backup from CentOS to Ubuntu completely breaks Webmin

### ISSUE THREAD

**Issue by lunarovic** [webmin/webmin#909]
Fresh Webmin installation on Ubuntu. After restoring configuration backup from CentOS, Webmin went completely dead, cannot even load a page.

I know there are big differences between the operating systems, so is it even possible to backup configuration on one OS and then restore it to another? If not, then there should be some kind of a flag in the config file to stop restoring if the backup is from a different OS.

**Comment by jcameron** [comment]
Which module's configs were you restoring exactly, and on what page were you doing it?

**Comment by lunarovic** [comment]
It's the "Backup Configuration Files" module, I used it both for backup and restore. I selected all modules for backup, I really can't name them specifically because there is a lot of them, but what's important is that I didn't install any extra or custom modules, those were only the default ones.

Now when I look in the module's options for backup, there is this:

> Include in backup:
> - [x] Webmin module configuration files
> - [x] Server configuration files
> - [ ] Other listed files ..

And I checked them like shown above. Totally wild guess, but maybe the _Server configuration files_ option breaks things when the new server is on a different OS? By the way, the exact OS is Ubuntu 16.04.4

Anyway, I saved the configuration backup file which broke my Webmin installation, and I just sent you an email with that file attached if you wish to try it on Ubuntu (and have the time to do so). But of course be careful, only try it in a test environment. 🙂

**Comment by jcameron** [comment]
Yeah, using that module to migrate configs between operating systems isn't supported, sorry. I really should add a warning in this case, as it can lead to breakage when the location and format of config files differ.
